src/Cache/Engine/NullEngine.php

```diff
@@ -62,7 +62,13 @@ public function get($key, $default = null)
      */
     public function getMultiple($keys, $default = null): iterable
     {
-        return [];
+        $result = [];
+
+        foreach ($keys as $key) {
+            $result[$key] = $default;
+        }
+
+        return $result;
```

Comment:
```suggestion
        return array_fill_keys($keys, $default);
```

Reply:
Keeping foreach since `$keys` is an iterable.